apps/api-extractor/package.json

```diff
@@ -52,8 +52,8 @@
   },
```

Comment:
The `cyclicDependencyProjects` setting was renamed to `decoupledLocalDependencies`.

- Let's update our `rush.json` settings
- Let's rename the PR title to "Bump decoupled dependencies" -- going forward, we'll use that naming

Reply:
That change in Rush hasn't been published yet.